src/css/index.scss

```diff
@@ -1,2 +1,356 @@
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  width: 100%;
+  padding: 20px 0 0 0;
+  height: 100px;
+  position: fixed;
+  z-index: 1;
+}
+
+.headleft {
+  width: 50%;
+  float: left;
+}
+
+.headright {
+  width: 50%;
+  float: right;
+}
+
+#home {
+  text-decoration: none;
+  font-size: 28px;
+  color: $primary_color;
+  font-weight: bold;
+  float: left;
+  margin-left: 10%;
+}
+
+#download {
+  margin-right: 10%;
+  float: right;
+  text-decoration: none;
+  border: 2px solid $primary_color;
+  font-size: 18px;
+  padding: 4px 25px;
+  color: $primary_color;
+  &:hover {
+    background: $btn_hover_color;
+  }
+}
+
+.skills-section {
+  position: relative;
+  width: 100%;
+  h2 {
+    color: $primary_color;
+    letter-spacing: 4px;
+    max-width: 949px;
+    margin: 0 auto;
+    &::after {
+      content: '';
+      display: block;
+      width: 100px;
+      border: 3px solid $btn_hover_color;
+      margin: 30px 0;
+    }
+  }
+}
+
+.skill-inner-width {
+  width: 949px;
+  margin: 0 auto 20px;
+  display: flex;
+}
+
+.skill-left,
+.skill-right {
+  height: 436px;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.skill-left {
+  width: 513px;
+}
+.skill-right {
+  width: 436px;
+}
+
+.skill {
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+}
+
+.skill_name,
+.skill_level {
+  flex: 1;
+}
+
+.skill_level {
+  font-weight: bold;
+}
+
+#msexcel,
+#msword {
+  width: 147px;
+  height: 147px;
+  order: 0;
+  .skill_name {
+    font-size: 10px;
+    transform: translateY(115%);
+  }
+  .skill_level {
+    font-size: 18px;
+    transform: translateY(-20%);
+  }
+}
+
+#adobe,
+#french {
+  width: 219px;
+  height: 219px;
+  .skill_name {
+    font-size: 15px;
+    font-weight: 500;
+    transform: translateY(115%);
+  }
+  .skill_level {
+    font-size: 25px;
+    transform: translateY(-20%);
+  }
+}
+
+#ppt {
+  width: 294px;
+  height: 294px;
+  order: 1;
+  transform: translateY(-75px);
+  .skill_name {
+    font-weight: 700;
+    transform: translateY(115%);
+  }
+  .skill_level {
+    font-size: 30px;
+    transform: translateY(-15%);
+  }
+}
+
+#spanish {
+  width: 438px;
+  height: 438px;
+  .skill_name {
+    font-size: 20px;
+    font-weight: 600;
+    transform: translateY(115%);
+  }
+  .skill_level {
+    font-size: 50px;
+    transform: translateY(-20%);
+  }
+}
+
+#msexcel,
+#ppt,
+#french {
+  background: #edeff5;
+  .skill_level {
+    color: #e14e5e;
+  }
+}
+#adobe,
+#msword,
+#spanish {
+  background: #bbc8dd;
+  .skill_level {
+    color: #2e4975;
+  }
+}
+
+#adobe {
+  order: 1;
+}
+#french {
+  order: 2;
+}
+
+.contacts-section {
+  border: 0 solid;
+  font-size: 0;
+  flex-flow: column wrap;
+  h2 {
+    letter-spacing: 4px;
+    &::after {
+      content: '';
+      display: block;
+      width: 100px;
+      border: 3px solid $btn_hover_color;
+      margin: 30px 0;
+    }
+  }
+}
+
+.contacts-section > section {
+  flex: 1;
+  width: 50%;
+  display: inline-block;
```

Comment:
Not a bit fun of display: inline-block; for layout, but it is not wrong.
maybe float is bit better, at least you don't have to set font-size: 0 on parent.